interfaces/vrep/robots/YouBotVrepRobot.m

```diff
@@ -17,11 +17,11 @@
 %       will become "youBot#0", a third robot, "youBot#1", and so on.
 %
 %   YouBotVrepRobot Methods:
-%       send_q_to_vrep - Sends the joint configurations to VREP
-%       get_q_from_vrep - Obtains the joint configurations from VREP
+%       set_configuration_space_positions - Sends the joint configurations to VREP
```

Comment:
Same as before: please change the name and deprecate legacy names.

Reply:
Updated in [8b53f6d](https://github.com/dqrobotics/matlab/pull/108/commits/8b53f6ddc4ffc68d5f36e70b509d0ff269dd3330).